index.html

```diff
@@ -2,9 +2,152 @@
+         <p>@EPIC all rights reserved 2022.Epic game. 2020</p>
+       </div>
+      </div>
+    </div>
+
+    <div class="wrapper2">
+     <h5 class="text-prog">Main Program</h5>
+     <div class="line"></div>
+     <div class="card-wrap">
+       <div class="cart">
+         <img src="images/gym1.png" alt="">
+         <h5>Exercises Mixed</h5>
+         <p class="text-light" >Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym2.jpg" alt="">
+         <h5>Mixed Programs</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym3.png" alt="">
+         <h5>Weekly program</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym4.png" alt="">
+         <h5>Months program</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+       <div class="cart">
+         <img src="images/gym5.jpg" alt="">
+         <h5>Yearly programs</h5>
+         <p class="text-light">Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>
+       </div>
+     </div>
+     <div class="btn-container">
+       <button type="button" class="button">Join The EpicGame Program 2022</button>
+     </div>
+    </div>
+    <div class="wrapper3">
+      <h5 class="text"> Feature Speakers</h5>
+      <div class="line"></div>
+      <div class="speaker-wrap">
+        <section class="speakers">
+          <img class="speaker-img" src="images/speaker1.jpg" alt="">
+          <div class="side-info">
+            <h5 class="title">a</h5>
+            <span class=" about text-danger"></span>
+            <div class="line-"></div>
+            <p class="disc"></p>
+          </div>
+        </section>
+        <section class="speakers">
+          <img class="speaker-img" src="images/speaker1.jpg" alt="">
+          <div class="side-info">
+            <h5 class="title">a</h5>
+            <span class="about text-danger"></span>
+            <div class="line-"></div>
+            <p class="disc"></p>
+          </div>
+        </section>
+        <section class="speakers">
+          <img class="speaker-img" src="images/speaker1.jpg" alt="">
+          <div class="  side-info">
+            <h5 class="title">a</h5>
+            <span class="about text-danger"></span>
+            <div class="line-"></div>
+            <p class="disc"></p>
+          </div>
+        </section>
+        <section class="speakers">
+          <img class="speaker-img" src="images/speaker2.jpg" alt="">
+          <div class="side-info">
+            <h5 class="title">a</h5>
+            <span class="about text-danger"></span>
+            <div class="line-"></div>
+            <p class="disc"> </p>
+          </div>
+        </section>
```

Comment:
- [ ] According to the requirements the whole section (Speakers) should be created dynamically using the information stored in that JavaScript object. So, kindly remove all of the HTML in that section and create them dynamically using JS when the page loads. 👌

- I noticed you already generated the content of the title and description dynamically, but you should generate the entire HTML for this section dynamically. so I suggest removing all the HTML code from here and creating it using Js 

![image](https://user-images.githubusercontent.com/45231276/181515920-e6e0a85e-6a17-4f4d-a11b-620b28f82156.png)